tools/didc-js/README.md

```diff
@@ -1,68 +1,3 @@
 # @dfinity/didc
 
 A multi-purpose Candid tool for JavaScript projects, including encoding and decoding Candid values. 
-
-## Usage
```

Comment:
Why removing the usage?

Reply:
Because that's already included inside [wasm-package/README.md](https://github.com/dfinity/candid/pull/578/files#diff-991b05c0305831073a0d082d60b25cd03ab1379cf2299fa907d356dae484e688) which is the one that get's published to npmjs.